README.md

```diff
@@ -1,34 +1,80 @@
-# Git Repository Template
+# React App
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+This project was bootstrapped with [Nimble React template](https://github.com/nimblehq/react-templates).
 
-## Usage
+## Available Scripts
```

Comment:
I know our template doesn't have the `Prerequisites` section, but we should add it to tell other people which Node version we should use to run this project. The recommended version is the current LTS Node version.